insertUrl.py

```diff
@@ -0,0 +1,97 @@
+import sqlite3
+import read_lyrics
+
+
+class databaseConnect:
+    def __init__(self,dbFile):
+        self.conn = sqlite3.connect(dbFile)
+        self.c = self.conn.cursor()
+
+    def creatURLDB(self,url, url2, listWords):
+        self.c.execute('SELECT * FROM urldb WHERE url=?', (url,))
+        if self.c.fetchone() == None:
+            # Create table
+            # Insert a row of data
+            self.c.execute("CREATE TABLE " + url2 + " (ID Integer, Words text, Color Text)")
+            self.c.execute("INSERT INTO urldb VALUES (?,?)", (url, 0))
+            for i in range(len(listWords)):
+                self.c.execute("INSERT INTO " + url2 + " VALUES (?,?,?)", (i, listWords[i], 'NA'))
+
+            # self.c.execute("INSERT INTO " + url2 + " VALUES (?,?,?)", (1, 'sample text word', 'sample text color'))
+            # self.c.execute("INSERT INTO " + url2 + " VALUES (?,?,?)", (2, 'sample text word', 'sample text color'))
```

Comment:
Left some commented-out test code in here - not the best practices, but ah well.